### ISSUE TITLE
How to advertise ingred's actions?

### ISSUE THREAD

**Issue by lime62** [beatsbucket/mashub#6]
Need to find effective way.

**Comment by wingman7** [comment]
No need to advertise all details. Just advertise name/msg type(kind of protocol)/description.

**Comment by lime62** [comment]
Right. Also, actions should have type(readable/writable) itself. Readable actions is for monitoring and writable actions is for execution.
